warehouse/app/warehouse/__init__.py

```diff
@@ -74,13 +74,15 @@ def _configure_root_dir():
         """Create root dir if not exist and set root in `sftp.Handler`
         :return:
         """
-        root = os.getenv('ROOT_PATH', '/zim_files')
-
-        if os.path.exists(root):
-            if not os.path.isdir(root):
-                os.remove(root)
-                os.makedirs(root)
-        else:
-            os.makedirs(root)
+        root = os.getenv('ROOT_PATH', '/files')
+
+        for subdir_name in ('zim', 'logs'):
+            subdir = os.path.join(root, subdir_name)
+            if os.path.exists(subdir):
+                if not os.path.isdir(subdir):
+                    os.remove(subdir)
```

Comment:
I am not sure, but if the subdir has content in it, is `remove` gonna be successful?

Reply:
According to the [docs](https://docs.python.org/3.7/library/os.html#os.remove), `rmdir` is needed to remove a dir